authors/apps/notifications/views.py

```diff
@@ -0,0 +1,46 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework import status
+from authors.apps.notifications.models import Notification, Notifier
+from rest_framework.permissions import IsAuthenticated
+
+class NotificationViews(APIView):
+    permission_classes = (IsAuthenticated,)
+    def get(self, request, username):
+        current_user = request.user
+        notifications = Notification.objects.filter(receiver=current_user)
+        all_notifications = []
+        print(notifications)
```

Comment:
Great work, however, noticed a `print()` statement, probably must have used it during troubleshooting and forgot it, please remove it.